src/main/kotlin/org/rust/ide/refactoring/move/common/RsMoveCommonProcessor.kt

```diff
@@ -391,6 +392,28 @@ class RsMoveCommonProcessor(
         }
     }
 
+    fun updateMovedItemVisibility(item: RsItemElement) {
+        when (item.visibility) {
+            is RsVisibility.Private -> {
+                if (conflictsDetector.itemsToMakePublic.contains(item)) {
+                    val itemName = item.name
+                    val containingFile = item.containingFile
+                    if (item !is RsNameIdentifierOwner) {
+                        LOG.error("Unexpected item to make public: $item")
```

Comment:
How's about `RsUseItem`? Is it possible here?

Reply:
No, because this function is called on moved elements, and `RsUseItem` [can't be moved](https://github.com/intellij-rust/intellij-rust/blob/25fb0de04259de983046206d689b988c6f95f159/src/main/kotlin/org/rust/ide/refactoring/move/RsMoveTopLevelItemsHandler.kt#L136)